getting_started/brainvision.md

```diff
@@ -50,24 +50,26 @@ For validation of BrainVision file triplets, you can use the [brainvision-valida
 
 ## Preprocessing of raw EEG data
 
-FieldTrip needs the user to define what file to read in. The BrainVision Recorder software usually stores different filetypes (.vhdr, .eeg, .vmrk). For reading the data into FieldTrip you can refer to the .eeg file, for exampl
+FieldTrip needs the user to define what file to read in. The BrainVision Recorder software usually stores different filetypes (.vhdr, .dat, .vmrk). For reading the data into FieldTrip you can refer to the .dat file, for example. In order for FieldTrip to read the data file, it has to be exported as a .dat file from BrainVision.
```

Comment:
I think that the '.eeg' should stay here: this seems to be the common file extension, although I can imagine that exporting of partially processed data replaces the '.eeg' extension with '.dat'. May I suggest to change it back to '.eeg' here, because this section deals with raw data, and not with processed/segmented data yet.

Reply:
Ok!